### PR TITLE
Use case arguments

### PR DESCRIPTION
**Pull request by jadie1**
To address issue #760 I made the following changes:

- Changed `--start_with_prepped_data` to `--skip_grooming`
- Changed `--start_with_image_and_segmentation_data` to `--groom_images`
- Updated RunUseCases.py to print a warning if  `--groom_images` is used with a use case that does not have images.
- Updated the use case documentation for these changes on the mkdocs branch. 

To test run the use cases with these new tags.

### CODE REVIEW

File: Examples/Python/femur.py [hunk 1]
```diff
@@ -38,7 +38,16 @@ def Run_Pipeline(args):
     if args.interactive:
         input("Press Enter to continue")
 
-    if not args.start_with_prepped_data:
+    if args.skip_grooming:
```
Comment (sheryjoe) [inline Examples/Python/femur.py:41]: If getting the distance transforms file list is the same for all use cases, it is worth modularizing it into a utils function. For example, get_file_list with arguments directory, extension, is_tiny_test, num_samples.

File: Examples/Python/femur_cut.py [hunk 2]
```diff
@@ -39,7 +39,19 @@ def Run_Pipeline(args):
     if args.interactive:
         input("Press Enter to continue")
 
-    if not args.start_with_prepped_data:
+    if args.skip_grooming:
```
Comment (sheryjoe) [inline Examples/Python/femur_cut.py:42]: same comment about a function for file list.

File: Examples/Python/left_atrium.py [hunk 3]
```diff
@@ -50,7 +50,7 @@ def Run_Pipeline(args):
         fileList_seg= [fileList_seg[i] for i in sample_idx]
         fileList_img = [fileList_img[i] for i in sample_idx]
         
-    if args.start_with_prepped_data:
+    if args.skip_grooming:
```
Comment (sheryjoe) [inline Examples/Python/left_atrium.py:53]: Same here.

**Review by sheryjoe — CHANGES_REQUESTED** [review]

File: Examples/Python/RunUseCase.py [hunk 4]
```diff
@@ -61,6 +61,13 @@
 # make sure the shapeworks executables can be found
 robustifyShapeworksPaths()
 
+image_use_cases = ['femur', 'femur_cut', 'left_atrium']
+if args.groom_images and args.use_case.lower() not in image_use_cases:
```
[the remark this answers is not in the document]
Comment (jadie1) [reply Examples/Python/RunUseCase.py:65]: Agreed

**Comment by jadie1** [comment]
I made a common function for getting prepped data and addressed all of your comments @sheryjoe
I updated the documentation on the makedocs branch and tested all the use cases.

**Review by sheryjoe — CHANGES_REQUESTED** [review]
get_data (which downloads data) and get_groomed_data (which retrieves the file list in a directory) in CommonUtils do not have function names that reflect their functionality and it seems confusing.

**Comment by jadie1** [comment]
I changed "get_data" to "download_and_unzip_dataset" and I changed "get_groomed_data" to "get_distance_transform_list". Let me know if you can think of a better name for these functions.

**Comment by sheryjoe** [comment]
@jadie1 I don't think that get_distance_transform_list has anything specific to distance transforms. I would make this as a general get_file_list function with arguments for the extension to search for and the path to search in.

**Review by sheryjoe — APPROVED** [review]

**Review by akenmorris — APPROVED** [review]
Looks good.

**Pull request merged**